.travis.yml

```diff
@@ -2,14 +2,8 @@ language: go
 go_import_path: github.com/RHsyseng/operator-utils
 
 go:
-  - 1.10.x
   - 1.11.x
   - 1.12.x
 
 env:
-  - DEP_RELEASE_TAG="v0.5.1"
-
-before_install:
-  # Setup dependency management tool
-  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-  - dep version
+  - GO111MODULE=on
```

Comment:
add this to ensure vendor dir used for tests, vet, builds, etc...
`- GOFLAGS=-mod=vendor`